.github/workflows/int-and-connected-test-run.yml

```
@@ -59,6 +60,11 @@ jobs:
           secretId: ${{ secrets.SECRET_ID }}
```

```suggestion
          roleId: ${{ secrets.INTEGRATION_ROLE_ID }}
          secretId: ${{ secrets.INTEGRATION_SECRET_ID }}
```
could you try using these vault app role credentials instead? I just added them, they should have the same perms as what you had before plus access to buffer's vault paths